src/main/java/constants/OnlinerConstants.java

```diff
@@ -1,4 +1,4 @@
-package by.onliner.constants;
+package constants;
```

Comment:
Why have you updated this?

Reply:
Because i have problems with idea, and i want to update my packages

Reply:
Fixed